docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md

```diff
@@ -21,7 +21,7 @@ ms.assetid: 7536af08-4e86-4953-98a1-a8298623df92
 author: "rpetrusha"
 ms.author: "ronpet"
 ---
-# How to: Verify that strings are in valid email format
+# How to: verify that strings are in valid email format
```

Comment:
Verify should still be capitalized since it comes after a colon.

Reply:
@gewarren, The guideline was changed.

Reply:
#14179

Reply:
This is news to me. The Microsoft style guide says:

Sentence-style capitalization in titles and headings
Use sentence-style capitalization in most titles and headings: capitalize the first word and lowercase the rest. 
Exceptions Proper nouns, including brand, product, and service names, are always capitalized. If a title or heading includes a colon, capitalize the first word after it.

@mairaw Can you clarify?

Reply:
As the guideline was updated again in #15544, I'm editing the header to conform to the current guideline.